Clinical-Genomics/scout
# Redirect to variants page with message when variant is not found

Fix #2448 

**How to test**:
1. On scout stage (or local instance connected to LoqusDB), go to a variant (SV, SNV, STR) and copy the browser URL.
2. Paste this url omitting some characters, or replace the variant ID with a string of choice
3. Note that the page crashed
4. Repeat after installing this branch and make sure that page redirects with error without crashing

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
